Modules/mmapmodule.c

```diff
@@ -502,7 +502,7 @@ mmap_resize_method(mmap_object *self,
         CloseHandle(self->map_handle);
         /* if the file mapping still exists, it cannot be resized. */
         if (self->tagname) {
-            self->map_handle = OpenFileMapping(FILE_MAP_WRITE, FALSE,
+            self->map_handle = OpenFileMappingA(FILE_MAP_WRITE, FALSE,
```

Comment:
This works? I'd expect to have to skip building `mmap` when targeting `MS_WINDOWS_NON_DESKTOP`. In the 10.0.18362 SDK, `OpenFileMappingA()` and `CreateFileMappingA()` in "winbase.h" are limited to the desktop API partition, and `MapViewOfFile()` in "memoryapi.h" is limited to the desktop and system API partitions.

Reply:
Hm I have `10.0.19041` which provides it for desktop and games, but not for system
```
#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP | WINAPI_PARTITION_GAMES)
```

Reply:
I don't know whether building 3.12 will require Visual Studio 2019 with the 10.0.18362 SDK, or maybe Visual Studio 2022 with the 10.0.19041 SDK. Nothing has been updated yet regarding the requirements in "PCbuild/readme.txt" (still Visual Studio 2017) and "PC/pyconfig.h" (still Windows 8).

The ANSI file-mapping functions aren't available for the app API partition. In principle, building `mmap` should be skipped if `defined(MS_WINDOWS_NON_DESKTOP) && !defined(MS_WINDOWS_GAMES)`. Though it's not an urgent problem that needs to be resolved here.

Reply:
https://github.com/python/cpython/pull/102256/commits/b119bf5f439d41ad9c764265827e4f1ad49a3bae should skip the build.